src/python_bluesky_taskgraph/tasks/functional_tasks.py

```diff
@@ -63,7 +63,7 @@ class PlanTask(BlueskyTask["PlanTask.PlanInputs"]):
 
     @dataclass
     class PlanInputs(Input):
-        args: List[Any]
+        args: List[Any] = field(default_factory=list)
```

Comment:
Ditto, does this need to be updated wrt `TypedInput`?

Reply:
See above

Most plans aren't single argument. PlanTask going to face big refactor when I approach it likely.